libdnf5/module/module_sack.cpp

```diff
@@ -459,12 +463,12 @@ void ModuleSack::Impl::enable_dependent_modules() {
 }
 
 
-std::pair<std::vector<std::vector<std::string>>, ModuleSack::ModuleErrorType> ModuleSack::Impl::module_solve(
-    std::vector<ModuleItem *> module_items) {
-    std::vector<std::vector<std::string>> problems;
+std::pair<std::vector<std::vector<std::tuple<ProblemRules, Id, Id, Id, std::string>>>, GoalProblem>
+ModuleSack::Impl::module_solve(std::vector<ModuleItem *> & module_items) {
+    std::vector<std::vector<std::tuple<ProblemRules, Id, Id, Id, std::string>>> problems;
```

Comment:
I think it wouldn't be clear what the `{}` means for the `std::make_pair`.  If only the `problems` were returned, then it would work, because it would mean to return the defaultly initialized object of the return type, but since a pair must be constructed first, this won't work.